src/test/java/com/github/tomakehurst/wiremock/stubbing/StubMappingJsonRecorderTest.java

```diff
@@ -421,7 +421,7 @@ public void decompressesGzippedResponseBodyAndRemovesContentEncodingHeader() {
           + "		\"url\": \"/multipart/content\",							\n"
           + "		\"multipartPatterns\" : [ {									\n"
           + "			\"name\" : \"binaryFile\",								\n"
-          + "			\"matchingType\" : \"ALL\",								\n"
+          + "			\"matchingType\" : \"ANY\",								\n"
```

Comment:
Is this change really needed?

Reply:
looking

Reply:
Yes, there's an example of mis-match in `src/test/java/com/github/tomakehurst/wiremock/matching/RequestPatternTest.java#doesNotMatchExactlyWhenManyMultipartPatternsSupposeToMatchAllDistinctParts()`